packages/css/src/components/form/form-groups/search-bar/form-group-search-bar.template.ts

```diff
@@ -0,0 +1,57 @@
+import { FormGroupSearchBar } from '@dso-toolkit/sources';
+
+import { html, nothing, TemplateResult } from 'lit-html';
+import { classMap } from 'lit-html/directives/class-map.js';
+import { ifDefined } from 'lit-html/directives/if-defined.js';
+
+import { infoButtonTemplate } from '../../../info-button/info-button.template';
+import { infoTemplate } from '../../../info/info.template';
+import { searchBarTemplate } from '../../../search-bar/search-bar.template';
+
+export function formGroupSearchBarTemplate(formGroup: FormGroupSearchBar<TemplateResult>): TemplateResult {
+  const errorTextId = `${formGroup.id}-error-text`;
+  const helpTextId = `${formGroup.id}-help-text`;
+  const infoTextId = `${formGroup.id}-info-text`;
+
+  const ariaDescribedBy = [
+    formGroup.info?.fixed ? infoTextId : undefined
+  ]
+    .filter(s => !!s)
+    .join(' ') || undefined;
+
+  return html`
+    <div
+      class="form-group dso-filter ${classMap({ [`dso-${formGroup.state}`]: !!formGroup.state })}"
+      aria-describedby=${ifDefined(ariaDescribedBy)}
```

Comment:
In de Nunjucks staat `aria-describedby` alleen op de `@search-bar`, waarom heb je 'm hier neergezet?

Reply:
Goed punt, deze is hier helemaal niet nodig. Ik denk dat ik me vergist heb